concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1326,10 +1326,10 @@ Hale-1973-1798-1324	Human_Mind_and_Conditions-212	worship idols	v	10b3.22	257	WO
 Hale-1973-1798-1325	Human_Mind_and_Conditions-213	pray	v	10b3.23	24	PRAY
 Hale-1973-1798-1326	Human_Mind_and_Conditions-214	believe	v	10b3.24	1890	BELIEVE
 Hale-1973-1798-1327	Human_Mind_and_Conditions-215	offer to gods	v	10b3.25		
-Hale-1973-1798-1328	Human_Mind_and_Conditions-216	sacrifice	v	10b3.26		
+Hale-1973-1798-1328	Human_Mind_and_Conditions-216	sacrifice	v	10b3.26	1103	SACRIFICE
```

Comment:
Would unmap, since this is a noun in concepticon: https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L1104